pkg/event/message/headers.go

```diff
@@ -0,0 +1,46 @@
+package message
+
+import (
+	"fmt"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+type EventHeaderKey string
+
+const (
+	HdrType                 EventHeaderKey = "Type"
+	HdrXRhIdentity          EventHeaderKey = "X-Rh-Identity"
+	HdrXRhInsightsRequestId EventHeaderKey = "X-Rh-Insights-Request-Id"
+)
+
+type Header map[EventHeaderKey]string
+
+const (
+	HdrTypeIntrospect = "Introspect"
+)
+
+func (h *Header) Set(key EventHeaderKey, value string) {
+	var m map[EventHeaderKey]string = *h
+	m[key] = value
+}
```

Comment:
Are the Header methods here (Set, Get, GetAll) used anywhere? I can't references find them.

Reply:
thanks! I have to clean-up this!